.github/workflows/goreleaser.yml

```diff
@@ -61,6 +61,6 @@ jobs:
     - name: Package and push chart
       run: |
         make chart IMG=${{ env.REGISTRY }}/${{ env.OPERATOR_IMAGE }}:${{github.ref_name}}
-        helm package ${{ env.CHART_PATH }} --destination . --version ${{github.ref_name}}
-        helm push ./*.tgz oci://${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}    
+        helm package ${{ env.CHART_PATH }} --destination . --version ${{github.ref_name}} --app-version ${{github.ref_name}} --dependency-update
```

Comment:
@francostellari do we need the dependency update even if we do not use the sub-chart ?